src/sui/themes/rtd-site/globals/site.overrides

```diff
@@ -74,6 +74,7 @@ body {
   }
   a,
   a:hover {
+    color: @footerHoverColor;
```

Comment:
This is also on the `a` selector, so will alter the default link color. Hover would not be noticeable in this case, so should probably have a dedicated selector.

Reply:
👍  Ups, good catch! That was not the intention. https://github.com/readthedocs/site-community/pull/68/commits/dd7d9232afe4a05a213e15aaba0438ca04b4de3a should fix.